hedyorg/hedy
# [FIX] Scripts are broken on slides page

The JavaScript was not being initialized on the slides page.

Fix that by extending `layout.html`, where initialization happens, and conditionally hiding the relevant bits of UI we don't want.

Add a test to confirm.

**How to test**

Go to http://localhost:8080/adventure/story/1/raw and observe that the code is syntax highlighted.
